internal/engine/build_and_deployer.go

```diff
@@ -68,9 +68,10 @@ func (composite *CompositeBuildAndDeployer) BuildAndDeploy(ctx context.Context,
 			return store.BuildResultSet{}, err
 		}
 
-		if _, ok := err.(RedirectToNextBuilder); ok {
-			logger.Get(ctx).Debugf("(expected error) falling back to next build and deploy method "+
-				"after error: %v", err)
+		if redirectErr, ok := err.(RedirectToNextBuilder); ok {
+			s := fmt.Sprintf("(expected error) falling back to next build and deploy method "+
```

Comment:
"(expected error)" strikes me as odd phrasing. Most of these don't seem like an error to a user? (Know it's not edited in this, but I think it's worth thinking about now that we're showing it to more people)